src/Init/Data/Char/Tables.lean

```diff
@@ -0,0 +1,48 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jean-Baptiste Tristan
+-/
+-- DO NOT EDIT: file generated using the scripts/unicode tool
+prelude
+import Init.Data.Char.UnicodeSkipList
+import Init.Data.Array.BasicAux
+
+namespace Char.UnicodeSkipList
+instance numericTable : UnicodePropertyTable where
```

Comment:
Please regenerate the table after your latest change to the generation script changing `instance` to `def`.